yb-voyager/cmd/import.go

```diff
@@ -206,8 +206,8 @@ func registerImportDataCommonFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&tableListFilePath, "table-list-file-path", "",
 		"path of the file containing the list of the source db table names to import data")
 
-	BoolVar(cmd.Flags(), &tconf.EnableUpsert, "enable-upsert", true,
-		"Enable UPSERT mode on target tables")
+	BoolVar(cmd.Flags(), &tconf.EnableUpsert, "enable-upsert", false,
```

Comment:
We use this Flag for streaming in Oracle targetDB for upserting any insert and marking this false here itself will not be a good idea.